mantidproject/mantid
# Remove convert to point data from SofQWMoments

**Description of work.**

This PR removes a use of the convert to point data from the SofQWMoments algorithm. Fixing an issue that stopped it working with OSIRIS data.

**Report to:** @SpencerHowell

**To test:**

Run the following and check that the output `_moments` workspace is not NaN.

```
from mantid.simpleapi import *

ISISIndirectEnergyTransferWrapper(InputFiles=r'\\isis.cclrc.ac.uk\inst$\ndxosiris\instrument\data\cycle_20_2\OSIRIS00144076.nxs', Instrument='OSIRIS', Analyser='graphite', Reflection='002', Efixed=1.845, SpectraRange='963,1004', FoldMultipleFrames=False, GroupingMethod='Individual', OutputWorkspace='osiris144076_graphite002_Reduced')
ConvertSpectrumAxis(InputWorkspace='osiris144076_graphite002_red', OutputWorkspace='osiris144076_graphite002_rqw', Target='ElasticQ', EMode='Indirect')
SofQW(InputWorkspace='osiris144076_graphite002_red', OutputWorkspace='osiris144076_graphite002_sqw', QAxisBinning='0.2,0.05,1.75', EMode='Indirect', EFixed=1.845, ReplaceNaNs=True, Method='NormalisedPolygon')
AddSampleLog(Workspace='osiris144076_graphite002_sqw', LogName='rebin_type', LogText='NormalisedPolygon')
SofQWMoments(InputWorkspace='osiris144076_graphite002_sqw', EnergyMin=-0.61040899999999998, EnergyMax=0.68986399999999992, OutputWorkspace='osiris144076_graphite002_Moments')
```

Fixes #33777

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
